lingetic-nextjs-frontend/app/components/questions/FillInTheBlanks/WordExplanationHover.tsx

```
@@ -23,15 +25,15 @@ export default function WordExplanationHover({
 
   return (
     <span
-      className="relative cursor-pointer px-2 py-1 rounded bg-skin-fill-accent hover:bg-skin-fill-accent/70 transition-colors mx-1"
+      className={`cursor-pointer ${className ?? ""}`}
       ref={refs.setReference}
       onMouseEnter={() => setOpen(true)}
       onMouseLeave={() => setOpen(false)}
       tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element span is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)